posthog/api/test/test_event.py

```diff
@@ -61,9 +61,8 @@ def test_filter_events_by_event_name(self):
         _create_event(event="event_name", team=self.team, distinct_id="2", properties={"$ip": "8.8.8.8"})
         _create_event(event="another event", team=self.team, distinct_id="2", properties={"$ip": "8.8.8.8"})
         flush_persons_and_events()
-
         expected_queries = (
-            6  # Django session, PostHog user, PostHog team, PostHog org membership, person and distinct id
+            10  # Django session, PostHog user, PostHog team, PostHog org membership, person and distinct id, 4x license
```

Comment:
Why is license being fetched 4x in one request now?

Reply:
That's what `team.person_on_events_querying_enabled` does. Every time we make a query with hogql "properties.bla" in it, we make a request to check async migration, the license, etc. This should all be cached in production, but there's an explicit `not TEST` in some of the caching code.